Build failure on nightly-2018-02-01

Clippy seems to be failing to compile on nightly-2018-02-01 with this error:

```
error[E0063]: missing field `approximate` in initializer of `rustc_errors::CodeSuggestion`
   --> clippy_lints\src\utils\mod.rs:636:16
    |
636 |     let sugg = rustc_errors::CodeSuggestion {
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `approximate`
```

Presumably caused by rust-lang/rust#47540

--- a/clippy_lints/src/utils/mod.rs
+++ b/clippy_lints/src/utils/mod.rs
@@ -648,6 +648,7 @@ where
         ],
         msg: help_msg,
         show_code_when_inline: true,
+        approximate: false,
     };
     db.suggestions.push(sugg);
 }

